hoomd/md/pair.py

```diff
@@ -1414,72 +1176,57 @@ class moliere(pair):
                                 = & 0 & r > r_{\mathrm{cut}} \\
         \end{eqnarray*}
 
-    See :py:class:`_Pair` for details on how forces are calculated and the available energy shifting and smoothing modes.
-    Use ``coeff.set`` to set potential coefficients.
-
-    The following coefficients must be set per unique pair of particle types:
+    Where each parameter is defined as:
 
     - :math:`Z_i` - *Z_i* - Atomic number of species i (unitless)
     - :math:`Z_j` - *Z_j* - Atomic number of species j (unitless)
     - :math:`e` - *elementary_charge* - The elementary charge (in charge units)
     - :math:`a_0` - *a_0* - The Bohr radius (in distance units)
+
+    See :py:class:`_Pair` for details on how forces are calculated and the available energy shifting and smoothing modes.
+    Use ``params`` property to set potential coefficients.
+
+    The following coefficients must be set per unique pair of particle types:
+
+    - :math:`q_i` - *qi* - :math:`q_i = Z_i e`
```

Comment:
In HOOMD's system of units, this should read `q_i = Z_i e / (4 \pi \epsilon_0)` (in charge units) - and similarly for `q_j`.

Reply:
Shouldn't that be `q_i = Z_i e / \sqrt{4 \pi \epsilon_0}` then?